packages/terra-abstract-modal/tests/wdio/abstract-modal-spec.js

```diff
@@ -150,6 +150,38 @@ Terra.describeViewports('Abstract Modal', ['medium'], () => {
         Terra.validates.element('focus on modal', { selector });
       });
     });
+
+    describe('Focus Trapped Content', () => {
+      it('clicks to open modal', () => {
+        browser.url('/raw/tests/terra-abstract-modal/abstract-modal/focus-trapped-abstract-modal');
+        $('button').click();
+      });
```

Comment:
This doesn't seem to be validating anything; is this intended to set up subsequent tests or is there a missing expect/validate statement?
```suggestion
      it('clicks to open modal', () => {
        browser.url('/raw/tests/terra-abstract-modal/abstract-modal/focus-trapped-abstract-modal');
        $('button').click();
      });
```

Reply:
This is just a setup step that doesn't validate anything

Reply:
We can use `beforeAll` or `beforeEach` instead.

```suggestion
      beforeAll(() => {
        browser.url('/raw/tests/terra-abstract-modal/abstract-modal/focus-trapped-abstract-modal');
        $('button').click();
      });
```